SimulationRuntime/cpp/Makefile.omdev.mingw

```diff
@@ -81,7 +81,7 @@ else
 	PARALLEL_OUTPUT_COMMAND=-DUSE_PARALLEL_OUTPUT=OFF
 endif
 
-USE_LOGGER="false"
+USE_LOGGER="true"
 ifeq ("$(USE_LOGGER)","true")
```

Comment:
Unfortunately this will not work. This will evetually become,
`ifeq (""true"","true")`
and will fail. Either get rid of quotes from the assignment or from the if condition. See also my comment on ticket:4773.

Reply:
The quotes are removed with: https://github.com/OpenModelica/OMCompiler/pull/2252